### ISSUE TITLE
No valid versions with the prefix "1.18" found.

### ISSUE THREAD

**Issue by michidk** [googlecloudrobotics/core#86]
When running the setup with
```bash
curl -fS "https://storage.googleapis.com/cloud-robotics-releases/run-install.sh" >run-install.sh
bash ./run-install.sh $PROJECT_ID
```

This fails with
```bash
Error: googleapi: Error 400: No valid versions with the prefix "1.18" found., badRequest

  on cluster.tf line 1, in resource "google_container_cluster" "cloud-robotics":
   1: resource "google_container_cluster" "cloud-robotics" {
```

**Comment by michidk** [comment]
Actually, I think this has nothing to do with this project but rather with my GCE setup.

**Comment by michidk** [comment]
Actually, I tried running this on a GCE instance, GC Shell and a local linux installation. I always get the same error:
```
No matching resource instances found.
Releasing state lock. This may take a few moments...
data.google_project.project: Refreshing state...
google_logging_project_exclusion.remote_access_exclusion: Refreshing state... [id=projects/cloud-robotics-319117/exclusions/remote-access-exclusion]
google_logging_project_sink.remote_access_sink: Refreshing state... [id=projects/cloud-robotics-319117/sinks/remote-access-sink]
google_project_service.iamcredentials: Refreshing state... [id=cloud-robotics-319117/iamcredentials.googleapis.com]
google_project_service.servicemanagement: Refreshing state... [id=cloud-robotics-319117/servicemanagement.googleapis.com]
google_project_service.cloudiot: Refreshing state... [id=cloud-robotics-319117/cloudiot.googleapis.com]
google_project_service.iam: Refreshing state... [id=cloud-robotics-319117/iam.googleapis.com]
google_project_service.cloudbilling: Refreshing state... [id=cloud-robotics-319117/cloudbilling.googleapis.com]
google_project_service.cloudresourcemanager: Refreshing state... [id=cloud-robotics-319117/cloudresourcemanager.googleapis.com]
google_project_service.dns: Refreshing state... [id=cloud-robotics-319117/dns.googleapis.com]
google_project_service.pubsub: Refreshing state... [id=cloud-robotics-319117/pubsub.googleapis.com]
google_logging_project_bucket_config.remote_access_bucket: Refreshing state... [id=projects/cloud-robotics-319117/locations/global/buckets/RemoteAccess]
google_service_account.robot-service: Refreshing state... [id=projects/cloud-robotics-319117/serviceAccounts/robot-service@cloud-robotics-319117.iam.gserviceaccount.com]
google_project_service.endpoints: Refreshing state... [id=cloud-robotics-319117/endpoints.googleapis.com]
google_project_service.compute: Refreshing state... [id=cloud-robotics-319117/compute.googleapis.com]
google_project_service.servicecontrol: Refreshing state... [id=cloud-robotics-319117/servicecontrol.googleapis.com]
google_project_service.container: Refreshing state... [id=cloud-robotics-319117/container.googleapis.com]
google_project_iam_member.robot-service-kubernetes: Refreshing state... [id=cloud-robotics-319117/roles/container.developer/serviceAccount:robot-service@cloud-robotics-319117.iam.gserviceaccount.com]
google_service_account.human-acl: Refreshing state... [id=projects/cloud-robotics-319117/serviceAccounts/human-acl@cloud-robotics-319117.iam.gserviceaccount.com]
google_project_iam_member.robot-service-datastore: Refreshing state... [id=cloud-robotics-319117/roles/datastore.user/serviceAccount:robot-service@cloud-robotics-319117.iam.gserviceaccount.com]
google_project_service.containerregistry: Refreshing state... [id=cloud-robotics-319117/containerregistry.googleapis.com]
google_project_service.logging: Refreshing state... [id=cloud-robotics-319117/logging.googleapis.com]
google_project_service.serviceusage: Refreshing state... [id=cloud-robotics-319117/serviceusage.googleapis.com]
google_project_iam_member.robot-service-logging: Refreshing state... [id=cloud-robotics-319117/roles/logging.logWriter/serviceAccount:robot-service@cloud-robotics-319117.iam.gserviceaccount.com]
google_project_iam_member.robot-service-storage: Refreshing state... [id=cloud-robotics-319117/roles/storage.objectAdmin/serviceAccount:robot-service@cloud-robotics-319117.iam.gserviceaccount.com]
data.google_iam_policy.robot-service: Refreshing state...
google_project_iam_member.robot-service-monitoring: Refreshing state... [id=cloud-robotics-319117/roles/monitoring.metricWriter/serviceAccount:robot-service@cloud-robotics-319117.iam.gserviceaccount.com]
google_project_iam_member.robot-service-tracing: Refreshing state... [id=cloud-robotics-319117/roles/cloudtrace.agent/serviceAccount:robot-service@cloud-robotics-319117.iam.gserviceaccount.com]
google_storage_bucket.robot: Refreshing state... [id=cloud-robotics-319117-robot]
google_compute_address.cloud_robotics: Refreshing state... [id=projects/cloud-robotics-319117/regions/europe-west3/addresses/cloud-robotics]
google_service_account_iam_policy.robot-service: Refreshing state... [id=projects/cloud-robotics-319117/serviceAccounts/robot-service@cloud-robotics-319117.iam.gserviceaccount.com]
google_storage_bucket_object.install_k8s_on_robot: Refreshing state... [id=cloud-robotics-319117-robot-install_k8s_on_robot.sh]
google_storage_bucket_object.setup_robot_image_reference: Refreshing state... [id=cloud-robotics-319117-robot-setup_robot_image_reference.txt]
google_storage_bucket_object.setup_robot: Refreshing state... [id=cloud-robotics-319117-robot-setup_robot.sh]
google_storage_bucket_object.setup_robot_crc_version: Refreshing state... [id=cloud-robotics-319117-robot-setup_robot_crc_version.txt]
google_endpoints_service.www_service: Refreshing state... [id=www.endpoints.cloud-robotics-319117.cloud.goog]
google_container_cluster.cloud-robotics: Creating...

Error: googleapi: Error 400: No valid versions with the prefix "1.18" found., badRequest

  on cluster.tf line 1, in resource "google_container_cluster" "cloud-robotics":
   1: resource "google_container_cluster" "cloud-robotics" {


terraform apply failed
```

**Comment by drigz** [comment]
Sorry for the slow response - I haven't had time to dig into this this week. Maybe cluster.tf's 1.18 needs to be bumped to 1.20 if GKE no longer offers 1.18. You could try that out with the deploy from source instructions: https://googlecloudrobotics.github.io/core/how-to/deploy-from-sources - a PR would be welcome if it works!

**Comment by sammyKhan** [comment]
Fixed in 03c3daf04cbc133d3c52e007107e15628e01f5ae.

**Comment by tiesus** [comment]
Getting the same error, but with version 1.24.  I follow this guide https://googlecloudrobotics.github.io/core/how-to/deploy-from-sources . Error occurs during step 5 

Deploy the cloud project.

` ./deploy.sh create [PROJECT_ID]`

**Error message**

Error: googleapi: Error 400: No valid versions with the prefix "1.24" found.
Details:
[
  {
    "@type": "type.googleapis.com/google.rpc.RequestInfo",
    "requestId": "0x7b7d9ce2fd782a6d"
  }
]
, badRequest

  on cluster.tf line 11, in resource "google_container_cluster" "cloud-robotics":
  11: resource "google_container_cluster" "cloud-robotics" {


terraform apply failed
